misc/maven-resolver/build.gradle

```diff
@@ -41,21 +43,7 @@ dependencies {
     implementation 'org.apache.maven.resolver:maven-resolver-transport-file:1.4.2'
     implementation 'org.apache.maven.resolver:maven-resolver-transport-http:1.4.2'
     implementation 'me.tongfei:progressbar'
-    testImplementation 'org.apache.maven:maven-model-builder:3.6.3' 
     testCompile 'org.testng:testng'
 }
 
-test {
```

Comment:
Still there is a packaging issue in maven resolver with jdk11 where we cannot run tests within module. For this i have created a separate issue to enable 
https://github.com/ballerina-platform/ballerina-lang/issues/26786